### PR TITLE
Small fixes to the startup docs

### PR DESCRIPTION
**Pull request by coilysiren**
### Time to review: __1 mins__

Fixes https://github.com/HHS/simpler-grants-gov/issues/1011

## Changes proposed

1. adds instructions about copying the `frontend/.env.local` file
2. modifies the instructs to be run from the top level directory

## Context for reviewers

Nava slack: https://nava.slack.com/archives/C05GQF7DP33/p1704746497341639

### CODE REVIEW

File: documentation/api/development.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
 
 ## Run the application
 
-1. In your terminal, `cd` to the `api` directory of this repo.
+1. Run `cp frontend/.env.development frontend/.env.local` to create your local copy of the frontend environment variables
```
Comment (SammySteiner) [inline documentation/api/development.md:28]: Considering this is specifically in the api development doc, I'm thinking this isn't necessary here. 
As an aside, we seem to have different conventions for the frontend and backend dev setup docs, and there is no reason for that.

File: documentation/api/development.md [hunk 2]
```diff
@@ -25,13 +25,13 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
...
 2. Make sure you have [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed & running.
-3. Run `make setup-local` to install dependencies
-4. Run `make init start` to build the image and start the container.
+3. Run `make -C api setup-local` to install dependencies
```
Comment (SammySteiner) [inline documentation/api/development.md:30]: As someone who has multiple tabs open in terminal for each project subdirectory, I should really get in the habit of using `-C` in my make commands instead 🤦 . thanks for adding this!

**Review by SammySteiner — APPROVED** [review]
Thanks for this!

File: documentation/api/development.md (hunk 1 above)
Comment (coilysiren) [reply documentation/api/development.md]: duly removed!

**Pull request merged**